webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const secondaryActionButtons = (actions = []) =>
```

Comment:
does it make sense to create a component for each one of those? it can in theory make the snapshots smaller?

Reply:
Do you mean something like:

```js
const ActionButton = ({action: {title, url}, ...props}) => 
  <Button href={url}>{title}</Button>;
const PrimaryActionButton = (action) => 
  <ActionButton action={action} bsSize="large" bsStyle="primary" />
const SecondaryActionButtons = (actions = []) => 
  actions.map(action => <ActionButton action={action} />)
```